common/src/uint/impls.rs

```diff
@@ -135,6 +135,16 @@ macro_rules! construct_uint {
                 res
             }
 
+            fn _from_le_slice(bytes: &[u8]) -> $name {
```

Comment:
Two things
1. Please remove the `_`... this is not python :-)
2. Please restore the the functions you removed; just remove the `le` part to assert it's the default.